tests/integration/serverless.yml

```diff
@@ -17,7 +17,6 @@ provider:
     DD_TRACE_ENABLED: true
     DD_API_KEY: ${env:DD_API_KEY}
     DD_TRACE_MANAGED_SERVICES: true
-    DD_CAPTURE_LAMBDA_PAYLOAD: true
```

Comment:
Why did you remove this?

Reply:
I inadvertently added it when I wrote the feature, but it includes all headers and payload data that is complex to mask out, so I removed it now.

I think my introduction of this is what led to the tests being in the state they're in now.